docs/docs/guides/core-types/media.md

```diff
@@ -3,7 +3,7 @@ import TabItem from '@theme/TabItem';
 
 # Logging media
 
-Weave supports logging and displaying video, images, and audio.
+Weave supports logging and displaying video, images, audio, PDFs, and CSVs.
```

Comment:
We don't need to call out CSVs specifically - unlike PDFs or video/images/audio we don't have special in-app viewing capabilities for CSVs, so that is just an example of our generic File support, where you can pass an arbitrary File in or out of an Op and have the full contents available for download.

(It would be reasonable and not difficult to add an in-app preview of a CSV file in the future, but it's not something we have planned work for.)